pom.xml

```diff
@@ -23,11 +23,11 @@
 
 		<!-- DEPENDENCIES VERSION -->
 		<jdk.version>17</jdk.version>
-		<cds.services.version>2.4.0</cds.services.version>
+		<cds.services.version>2.4.1</cds.services.version>
```

Comment:
Please also update Spring Boot to `3.1.6` above

Reply:
done